source/screens/DrcFFlashScreen.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Screen.hpp"
+#include "FlashUtils.hpp"
+
+class DrcFFlashScreen : public Screen
```

Comment:
You can call the class and file `DrcFullFlashScreen`. No need to abbreviate here.